verifying/verifying_options.go

```diff
@@ -1,31 +1,46 @@
 package verifying
 
-import "github.com/spacemeshos/post/shared"
+import (
+	"github.com/spacemeshos/post/config"
+	"github.com/spacemeshos/post/shared"
+)
 
 type option struct {
-	verifyFunc func(val uint64) bool
-
 	logger shared.Logger
+	// scrypt parameters for AES PoW
+	powScrypt config.ScryptParams
+	// scrypt parameters for labels initialization
+	labelScrypt config.ScryptParams
 }
 
-func (o *option) validate() error {
-	return nil
+func defaultOpts() *option {
```

Comment:
I think this is only used in a single place and can thereby just be inlined.

Reply:
It's actually used in two places.

Reply:
The benefit is that defaults are defined in a single place and cannot go out of sync in contrast to if they were defined twice.